sdk/python/kfp/client/client.py

```diff
@@ -156,7 +156,7 @@ def __init__(
     ) -> None:
         """Create a new instance of kfp client."""
         warnings.warn(
-            'This client only works with Kubeflow Pipeline v2.0.0-alpha.0 '
+            'This client only works with Kubeflow Pipeline v2.0.0-beta.1 '
```

Comment:
This change should have been within https://github.com/kubeflow/pipelines/pull/9112